### ISSUE TITLE
New Update Affect Firerate

### ISSUE THREAD

**Issue by senordope** [liantian-cn/logitech-pubg#65]
Firerate has been affected due to the new update. Guns shoot at a stuttered rate.

**Comment by thestranger00** [comment]
yes same issue with me .. its seems okay in real time but when you watch the replay it actually fire in slow rate and stutter ...

**Comment by ahiddenmessi** [comment]
Yeah it’s hard to use akm now
